fedora/Makefile

```diff
@@ -7,7 +7,7 @@ RELEASE := $(shell set -x; sed -ne '/^Release:/s/.*  *\(.*\)%{.*}.*/\1/p' $(DIR)
 SRPM    := jellyfin-web-$(subst -,~,$(VERSION))-$(RELEASE)$(shell rpm --eval %dist).src.rpm
 TARBALL :=$(NAME)-$(subst -,~,$(VERSION)).tar.gz
 
-epel-7-x86_64_repos := https://rpm.nodesource.com/pub_16.x/el/\$$releasever/\$$basearch/
+epel-7-x86_64_repos := https://rpm.nodesource.com/pub_20.x/nodistro/\$$basearch/
```

Comment:
I'm not sure that this will function, the nodesource current docs recommend installing an rpm, which adds a new repo in `/etc/yum.repos.d` that looks like this:
```
[nodesource-nsolid]
name=Nsolid Packages for Linux RPM based distros - $basearch
baseurl=https://rpm.nodesource.com/pub_20.x/nodistro/nsolid/$basearch
priority=8
enabled=1
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/NODESOURCE-NSOLID-GPG-SIGNING-KEY-EL

[nodesource-nodejs]
name=Node.js Packages for Linux RPM based distros - $basearch
baseurl=https://rpm.nodesource.com/pub_20.x/nodistro/nodejs/$basearch
priority=9
enabled=1
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/NODESOURCE-NSOLID-GPG-SIGNING-KEY-EL
```
I believe the missing gpgkey will be an issue when installing this rpm.

Reply:
On Fedora 38 I wasn't able to install this package nor the previous one using yum, was this already broken?

Reply:
Probably... fedora tends to be perpetually broken :roll_eyes: